docker/api/container.py

```diff
@@ -256,7 +256,10 @@ def create_container(self, image, command=None, hostname=None, user=None,
 
         .. code-block:: python
 
-            client.api.create_host_config(port_bindings={1111: ('127.0.0.1', 4567)})
+            client.api.create_host_config(port_bindings={1111: (
+                '127.0.0.1',
+                4567
+            )})
```

Comment:
This should probably look like this according to other places:
```suggestion
            client.api.create_host_config(
                port_bindings={1111: ('127.0.0.1', 4567)})
```